CHANGELOG.md

```diff
@@ -7,6 +7,9 @@
 ### Features
 - Add grants to materializations ([#168](https://github.com/dbt-labs/dbt-snowflake/issues/168), [#178](https://github.com/dbt-labs/dbt-snowflake/pull/178))
 
+### Under the Hood
+- Reformat overridden macro location of grants work to a apply_grants.sql file in snowflake ([#193](https://github.com/dbt-labs/dbt-snowflake/issues/193), [#192](https://github.com/dbt-labs/dbt-snowflake/pull/192))
```

Comment:
Could you pop this up to a new section, `1.3.0b1`?

(this will be much easier once we have `changie` running in this repo!)